### PR TITLE
List reports by organization

### PR DESCRIPTION
**Pull request by msanroman**
### Notes
To be merged after the migration.

### CODE REVIEW

File: packages/report-list/middleware.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ import { actionTypes as reportListActionTypes } from './reducer';
 
 export default store => next => (action) => { // eslint-disable-line no-unused-vars
   switch (action.type) {
-    case `user_${actionTypes.FETCH_SUCCESS}`:
+    case `profiles_${actionTypes.FETCH_SUCCESS}`:
       store.dispatch(actions.fetch({
         name: 'list_reports',
         args: {
-          userId: action.result.id,
+          organizationId: action.result[0].organizationId,
```
Comment (federicoweber) [inline packages/report-list/middleware.js]: Hey @msanroman was wondering: why we are wrapping the result in an array? Do you think we can directly return the result, given that we are consuming only the first element anyway?

**Comment by hharnisc** [comment]
Sorry for the noise on this PR -- the review bot wasn't configured to receive the webhook for submitted reviews. That's all fixed up now :ok_hand:

File: packages/report-list/middleware.js (hunk 1 above)
Comment (msanroman) [reply packages/report-list/middleware.js]: It's grabbing the `organizationId` from the first profile of the list. We're consuming the full profile list in the profile selector reducer 😛

Comment (federicoweber) [reply packages/report-list/middleware.js]: Oh right :man_facepalming: :smile:

**Pull request merged**